src/node.js

```diff
@@ -105,7 +115,53 @@ define([
     node.removeChild(childNode);
   }
 
+  /**
+   * Chrome: If a parent node has a CSS `line-height` when we apply the
+   * insertHTML command, Chrome appends a SPAN to plain content with
+   * inline styling replicating that `line-height`, and adjusts the
+   * `line-height` on inline elements.
+   *
+   * As per: http://jsbin.com/ilEmudi/4/edit?css,js,output
+   * More from the web: http://stackoverflow.com/q/15015019/40352
+   */
+  function removeChromeArtifacts(parentElement) {
+    function isInlineWithStyle(parentStyle, element) {
+      return isInlineElement(element) &&
+        element.style.lineHeight &&
+        window.getComputedStyle(element).lineHeight === parentStyle.lineHeight ?
+        NodeFilter.FILTER_ACCEPT :
+        NodeFilter.FILTER_SKIP;
+    }
+
+    var iterator = document.createNodeIterator(
+      parentElement,
+      NodeFilter.SHOW_ELEMENT,
+      // arguments to .bind(), starting from the second, are automatically
+      // prepended to the actual arguments when the function is called
+      isInlineWithStyle.bind(null, window.getComputedStyle(parentElement))
+    );
+    var emptySpans = Immutable.List();
+    var node;
+
+    while (node = iterator.nextNode()) {
```

Comment:
Oh yeah, I think it could, something like:

```
var nodes = Immutable.List(parentElement.querySelectorAll(inlineElementNames.join(',')));
nodes = nodes.filter(...);
```

Ideally the selector would select only nodes with a `style` attribute, so this would need to be tweaked. Your call.